src/Idris/Completion.hs

```diff
@@ -159,6 +159,7 @@ completeColour (prev, next) = case words (reverse prev) of
 completeCmd :: String -> CompletionFunc Idris
 completeCmd cmd (prev, next) = fromMaybe completeCmdName $ fmap completeArg $ lookupInHelp cmd
     where completeArg FileArg = completeFilename (prev, next)
+          completeArg ShellCommandArg = completeFilename (prev, next)
```

Comment:
This is the [same completion approach](http://git.haskell.org/ghc.git/blob/9fd87ef8a16fbbce35205ae63d75d239bb575ccc:/ghc/GHCi/UI.hs#l2996) that GHCi uses for `:!`. One particularly nice thing about it is that if you type `:! vim direc` in a location where there's a directory named `directory`, then tab-completion fills in the rest to give you `:! vim directory`.